slang-antlr/src/main/antlr4/com/sonarsource/slang/parser/SLang.g4

```diff
@@ -135,6 +160,10 @@ nativeBlock
   :  LBRACK (statementOrExpression semi)* (statementOrExpression semi?)? RBRACK 
   ;
 
+returnExpression
```

Comment:
This rule cannot simply be called 'return' because in ANTLR the 'symbol return conflicts with generated code in target language or runtime'